src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <img
```

Comment:
Please try to remove this image tags and try to use images like background for input. Because now I click on the icon on the input does not become in focus